src/storage/macros.rs

```diff
@@ -25,14 +27,26 @@
 /// define_intermediate!(1, Double -> i32, MyStorageType, |_: &Double, db: &DbHandle<MyStorageType>| {
 ///     db.get(MyInput) * 2
 /// });
+///
+/// // It is also possible to signal that the value always changes with the assume_changed keyword:
```

Comment:
Should leave a note here explaining this can help us avoid expensive `Eq` checks on large types which are likely to change any time the inputs change.